JamGame/Assets/Scripts/Level/Boss/Task.cs

```diff
@@ -11,7 +11,6 @@
 using Sirenix.OdinInspector;
 using UnityEngine;
 
-// TODO: Move all the DTOs to their source classes.
```

Comment:
Actually it's not important to have them near classes implementing `IDataProvide<...>`

Reply:
So, better hold them in one place?

Reply:
It's better to have a single approach across all the data providet DTOs, as they can be emitted from many places or consumed by many observers, so if we implement different approaches to these types of connections it'll be hard to find implementations